### ISSUE TITLE
"Nothing to upload, all files are on the server" when they are not

### ISSUE THREAD

**Issue by willdady** [getsentry/sentry-react-native#502]
OS:
- [ ]  Windows
- [*]  MacOS
- [ ]  Linux

_Platform:_
- [*]  iOS
- [ ]  Android

Output of `node -v && npm -v && npm ls --prod --depth=0`

```
v8.10.0
6.4.0
goodwork@1.0.1 /Users/dadyw/Documents/projects/goodwork-ample-app
├── @ptomasroos/react-native-multi-slider@0.0.14
├── assert@1.4.1
├── aws-sdk@2.135.0
├── crypto-js@3.1.9-1
├── hoist-non-react-statics@3.0.1
├── lodash@4.17.4
├── moment@2.22.0
├── path-parser@2.0.2
├── prop-types@15.6.0
├── query-string@6.1.0
├── UNMET PEER DEPENDENCY react@16.3.2 invalid
├── react-native@0.55.4
├── react-native-actionsheet@2.2.2
├── react-native-animatable@1.2.4
├── react-native-auth0@1.3.0
├── react-native-auto-height-image@1.0.1
├── react-native-communications@2.2.1
├── react-native-config@0.11.5
├── react-native-device-info@0.21.5
├── react-native-fast-image@5.1.0
├── react-native-fbsdk@0.8.0
├── react-native-firebase@4.3.5
├── react-native-gifted-chat@0.4.3
├── react-native-image-crop-picker@0.20.3
├── react-native-intercom@12.0.0
├── react-native-keyboard-aware-scroll-view@0.4.1
├── react-native-keychain@2.0.0-rc
├── react-native-permissions@1.0.1
├── react-native-screens@1.0.0-alpha.16
├── react-native-sentry@0.38.3
├── react-native-share@1.0.26
├── UNMET PEER DEPENDENCY react-native-svg@6.3.1
├── react-native-svg-uri@1.2.3
├── react-navigation@2.12.1
├── react-navigation-backhandler@1.1.1
├── react-navigation-tabs@0.8.4
├── react-redux@5.0.6
├── redux@3.7.2
├── redux-logger@3.0.6
├── redux-persist@5.2.2
├── redux-thunk@2.2.0
├── url@0.11.0
├── uuid@3.3.2
└── validate.js@0.12.0
```

Config:

```
Sentry.config('https://...@sentry.io/...').install()
```

---
I have following issue:

For some time now errors in Sentry have been missing source maps. I've spent the day trying to get them to work but it seems really inconsistent. I've got the source maps to appear under the release only once but can't replicate getting them to appear again after deleting them and also the debug files under _Settings > Debug Files_.

Despite there being no artifacts under the release and having removed the latest Debug Files I still get the following output when building:

> Found 2 debug information files
> Prepared debug information files for upload
> Nothing to upload, all files are on the server
> No new debug symbols to associate.

I've tried bumping the dist number and doing a clean build but no luck.

The **Bundle react native code & images** step looks like this:
```
export SENTRY_PROPERTIES=sentry.properties
export DEVELOPMENT_BUILD_CONFIGURATIONS="+(Dev|Debug)"
export NODE_BINARY=node
$NODE_BINARY ../node_modules/@sentry/cli/bin/sentry-cli react-native xcode ../node_modules/react-native-schemes-manager/lib/react-native-xcode.sh
```

The **Upload Debug Symbols to Sentry** step looks like this:
```
export SENTRY_PROPERTIES=sentry.properties
../node_modules/@sentry/cli/bin/sentry-cli upload-dsym --force-foreground 
```
I've also tried it with the following as suggested in #461:
```
export SENTRY_PROPERTIES=sentry.properties
../node_modules/@sentry/cli/bin/sentry-cli upload-dsym --force-foreground "${DWARF_DSYM_FOLDER_PATH}/${DWARF_DSYM_FILE_NAME}"
```
**Steps to reproduce:**
Build the app in XCode

**Actual result:**
No source maps appear under the release in Sentry

**Expected result:**
After changing code and rebuilding I expect debug files and source maps to upload to Sentry and appear under the release

**Comment by lorenzo-ange** [comment]
Same issue here. Did you manage to resolve?

**Comment by HazAT** [comment]
I am closing all old issues, please if this is still a problem feel free to revive it.
Also, consider moving to our new SDK `@sentry/react-native` if it still happens there please open a new issue.
